### DISCUSSION TITLE
Is product_id available in LineItem?

### DISCUSSION THREAD

**Discussion by radoslaw-sz** [medusajs/medusa#6303] [Q&A] [answered]
Hello,

According to documentation I see it shall be possible to get product_id from LineItem:
https://docs.medusajs.com/references/entities/classes/LineItem

Unfortunately, on the code level (LineItem in /models) product_id is not available in LineItem.

Is there any plan for update it? Or I am missing something?

**Comment by xyzones** [comment]
```typescript
  @FeatureFlagColumn(MedusaV2Flag.key, { nullable: true, type: "text" })
  product_id: string | null
```
v2 feature ...

**Comment by radoslaw-sz** [comment]
@xyzones  thanks! Do you know which flag is enabling it? I mean what flag shall I set? I cannot not find it here: https://docs.medusajs.com/beta

**Reply by olivermrbl** [reply, marked as answer]
The v2 flag is not documented, as it is currently under development and not ready for usage. We expect to announce a release candidate in ~3 months time. You can follow our roadmap to v2 [here](https://github.com/orgs/medusajs/projects/7).

**Reply by radoslaw-sz** [reply]
thanks for the info!
